pytorch3d/implicitron/dataset/json_index_dataset.py

```diff
@@ -65,7 +63,7 @@ class JsonIndexDataset(DatasetBase, ReplaceableBase):
     A dataset with annotations in json files like the Common Objects in 3D
     (CO3D) dataset.
 
-    Args:
+    Metadata-related args::
```

Comment:
This distinction between the two types of members doesn't matter to the user of this class; it's an implementation detail. We could leave as a big list in this comment, or we could split into groups like "Options affecting WHICH frames are loaded"/"Options affecting what data is loaded for each frame"/"Misc options".